ccd-definition/AuthorisationCaseState.json

```diff
@@ -6,6 +6,13 @@
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseStateID": "Deleted",
+    "UserRole": "caseworker-publiclaw-solicitor",
+    "CRUD": "R"
```

Comment:
Would it be possible to hide it from LA users so they do not see deleted applications?

Reply:
the problem with removing the R permission:

<img width="1159" alt="Screen Shot 2019-07-18 at 16 20 26" src="https://user-images.githubusercontent.com/29975020/61706913-38dd0880-ad41-11e9-8f3f-4341c4569b0c.png">